build/upload.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+# This file uploads the necessary build files to s3
+# which enables them to be served by our CDN.
+# To run locally, you must be logged into AWS,
+# have built the needed files and pass the tag.
+# To build files:
+# yarn run build-prod-min && yarn run build-prod && yarn run build-csp && yarn run build-dev && yarn run build-css
```

Comment:
This feels like a potential step that could be missed when running this script outside of CI. Is there any way to guarantee that the files that will be uploaded do in fact correspond to the tag specified?